java/KafkaConnectors/README.md

```diff
@@ -32,7 +34,85 @@ Configuration parameters:
 * `source.topic` source topic (default: `source`)
 * `sink.bootstrap.servers` sink cluster bootstrap servers
 * `sink.topic` sink topic (default: `destination`)
+* `sink.transaction.timeout.ms` Sink transaction timeout 
+
+If you are connecting with no-auth and no SSL, above will work. Else you need additional configuration for both source and sink.
+### For IAM Auth
+
+
+* `source.sasl.mechanism` AWS_MSK_IAM
+* `source.sasl.client.callback.handler.class` software.amazon.msk.auth.iam.IAMClientCallbackHandler
+* `source.sasl.jaas.config` "software.amazon.msk.auth.iam.IAMLoginModule required;"
+* `source.security.protocol` SASL_SSL
+* `source.ssl.truststore.location` /usr/lib/jvm/java-11-amazon-corretto/lib/security/cacerts
+* `source.ssl.truststore.password` changeit
+* `sink.sasl.mechanism` AWS_MSK_IAM
+* `sink.sasl.client.callback.handler.class` software.amazon.msk.auth.iam.IAMClientCallbackHandler
+* `sink.sasl.jaas.config` "software.amazon.msk.auth.iam.IAMLoginModule required;"
+* `sink.security.protocol` SASL_SSL
+* `sink.ssl.truststore.location` /usr/lib/jvm/java-11-amazon-corretto/lib/security/cacerts
+* `sink.ssl.truststore.password` changeit
+
 
-### Running locally in IntelliJ
+## Running locally in IntelliJ
 
 To start the Flink job in IntelliJ edit the Run/Debug configuration enabling *'Add dependencies with "provided" scope to the classpath'*.
+
+Provide arguments like following -
+```
+--source.bootstrap.servers localhost:9092 --source.topic source --sink.bootstrap.servers localhost:9092 --sink.topic sink --sink.transaction.timeout.ms 1000
```

Comment:
can you show how to add arguments in Intellij Environment properties? This is how most customers will do it and not run CLI with IDE

Reply:
These are just argument. 
There are two options to make it better - 
1. I will show the screenshot of run configurations. 
2. I can give the xml run configuration like this - 
<component name="ProjectRunConfigurationManager">
  <configuration default="false" name="SlidingWindowStreamingJobWithParallelism" type="Application" factoryName="Application" nameIsGenerated="true">
    <option name="INCLUDE_PROVIDED_SCOPE" value="true" />
    <option name="MAIN_CLASS_NAME" value="com.amazonaws.services.msf.windowing.kinesis.SlidingWindowStreamingJobWithParallelism" />
    <option name="PROGRAM_PARAMETERS" value="--InputStreamRegion ap-south-1 --InputStreamName stream-input --OutputStreamRegion ap-south-1 --OutputStreamName stream-output" />
    <extension name="coverage">
      <pattern>
        <option name="PATTERN" value="com.amazonaws.services.msf.windowing.kinesis.*" />
        <option name="ENABLED" value="true" />
      </pattern>
    </extension>
    <extension name="software.aws.toolkits.jetbrains.core.execution.JavaAwsConnectionExtension">
      <option name="credential" />
      <option name="region" />
      <option name="useCurrentConnection" value="false" />
    </extension>
    <method v="2">
      <option name="Make" enabled="true" />
    </method>
  </configuration>
</component>%

Reply:
I have added run configuration images. Does that work ?